schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts SMALLINT NOT NULL,
     neutered BOOLEAN NOT NULL,
     weight_kg FLOAT NOT NULL,
+    species VARCHAR(20) NOT NULL DEFAULT '',
```

Comment:
- [ ] Please, consider the previous reviewer's suggestion to use `ALTER TABLE` to add the `species` table. To avoid changing the previous query, you can do it by just adding into your `schema.sql` file the following code:
`ALTER TABLE animals ADD species varchar(255);`

In general, your `schema.sql` file will look like this:
```
CREATE TABLE animals (
    id INT GENERATED ALWAYS AS IDENTITY,
    name VARCHAR(20) NOT NULL,
    date_of_birth DATE NOT NULL,
    escape_attempts SMALLINT NOT NULL,
    neutered BOOLEAN NOT NULL,
    weight_kg FLOAT NOT NULL,
    PRIMARY KEY( id )
);

ALTER TABLE animals ADD species varchar(255);
```

Reply:
Why? is that more readable or maintainable, stated somewhere in the requirements?
one the other hand, it is segmenting a definition for no discernible reason.